server/src/termdb.matrix.js

```diff
@@ -372,7 +372,8 @@ export async function getSamplesPerFilter(q, ds) {
 	q.ds = ds
 	const samples = {}
 	for (const id in q.filters) {
-		const filter = filterJoin([q.filter, q.filters[id]])
+		let filter = q.filters[id]
+		if (!q.ignoreUserFilter && q.filter) filter = filterJoin([q.filter, q.filters[id]])
```

Comment:
this flag cannot be set by the client, it will allow a public role to sidestep restrictions on what they can see

Reply:
if a public user says he wants to ignore the user filter (this is not possible from the UI but lets say from the route) it will see the average of all the sites which is the default behaviour.  If he wants to apply the auth filter, that is based on the sites he has access to in the jwt, he will get an empty filter. Either way he wont see sensitive info.